<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatdemo</class>
 <widget class="QWidget" name="chatdemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chatdemo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowserContent">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFontComboBox" name="fontComboBoxFont">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxSize">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="currentText">
             <string>2</string>
            </property>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>18</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>22</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>26</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>28</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>32</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>34</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>36</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonColor">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonChatLog">
            <property name="text">
             <string>  聊天记录</string>
            </property>
            <property name="icon">
             <iconset resource="../../res.qrc">
              <normaloff>:/image/image/backup.png</normaloff>:/image/image/backup.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonManage">
            <property name="text">
             <string>  聊天管理</string>
            </property>
            <property name="icon">
             <iconset resource="../../res.qrc">
              <normaloff>:/image/image/administrator.png</normaloff>:/image/image/administrator.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEditSendText">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelInfo">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSend">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
